pkg/aws/eni/instances.go

```diff
@@ -141,6 +141,21 @@ func (m *InstancesManager) FindSecurityGroupByTags(vpcID string, required ipamTy
 	return securityGroups
 }
 
+// GetMatchingSubnets returns all subnets in the given VPC and availability zone.
```

Comment:
The commit message here refers to an ENU, which I can only assume is a hybrid of a large flightless bird and the network interfaces found deep in the jungles of the Amazon datacenters. Perhaps it was supposed to be ENI? ;-)